aliases/available/general.aliases.bash

```diff
@@ -57,11 +62,16 @@ alias ipy='ipython'
 alias piano='pianobar'
 
 alias ..='cd ..'         # Go up one directory
+alias cdh='cd ~/'  # Go in home
```

Comment:
I don't think this is necessary. You can simply run `cd` (without any parameters) and it should take you to your home directory.

Reply:
https://github.com/jusdepatate/bash-it/commit/cc91898eefb92c193880349bbbd548eda6efb784